src/components/common/Dialogs/AwardAndSmiteDialogs/ManageReputationContainer/helpers.ts

```diff
@@ -17,10 +19,26 @@ export const getManageReputationDialogPayload = (
     colonyAddress: colony.colonyAddress,
     colonyName: colony.name,
     domainId,
-    walletAddress: user.walletAddress,
+    userAddress: user.walletAddress,
     annotationMessage: annotation,
     amount: reputationChangeAmount.toString(),
     motionDomainId,
     isSmitingReputation: isSmiteAction,
   };
 };
+
+export const useGetColonyMembers = (colonyAddress?: Address | null) => {
+  const { data } = useGetMembersForColonyQuery({
```

Comment:
There's an implicit limit here of 100... 

Really we'd have a way of fetching more if you scroll to the bottom of the user picker (useful for colonies with more than 100 members).

But I think that's a "new UI" optimisation.